### PR TITLE
Add ansible-lint tooling added

### PR DESCRIPTION
**Pull request by ddimatos**
##### SUMMARY
This addresses issue #809 partially. There is still the remaining CI/CD work to be done that is not managed in this repository. We can continue to track the CI/CD work for ansible-lint using this git issue #809 but can consider this issue complete for now. 

##### ISSUE TYPE
- Feature Pull Request

### CODE REVIEW

**Comment by ddimatos** [comment]
There is no pipeline needed for this. @richp405 @ketankelkar

File: .ansible-lint [hunk 1]
```diff
@@ -0,0 +1,35 @@
+exclude_paths:
```
Comment (richp405) [inline .ansible-lint:1]: I don't know if this is significant, but I pulled up the docs, and .ansible-lint is supposed to be a titled yaml file: 
Probably minor detail: but here are the first 2 lines of the example:
---
# .ansible-lint

https://ansible.readthedocs.io/projects/lint/configuring/#specifying-configuration-files

File: galaxy.yml [hunk 2]
```diff
@@ -63,28 +63,33 @@ homepage: https://www.ibm.com/support/z-content-solutions/ansible/
 issues: https://github.com/ansible-collections/ibm_zos_core/issues
 
 # Ignore files and directories matching the following patterns
-build_ignore:
+  - '*.tar.gz'
```
Comment (richp405) [inline galaxy.yml:66]: I see where - build_ignore was removed, but not where it was added back in...

**Review by richp405 — CHANGES_REQUESTED** [review]
A missing intro line --- and a missing build_ignore.

File: .ansible-lint (hunk 1 above)
Comment (richp405) [reply .ansible-lint:1]: git ate my "---" on the first line.

Comment (ddimatos) [reply .ansible-lint:1]: Its a good point, but the docs also say it does not have to be.
We have a working `ansible-lint` on our playbook repo [here](https://github.com/IBM/z_ansible_collections_samples/blob/main/.ansible-lint)
Same docs you link to state this: 
```
Specify Ansible-lint configuration in either .ansible-lint, .config/ansible-lint.yml, or .config/ansible-lint.yaml in your current working directory.
```
and I am glad to see the docs up, many read-the-docs are down due to a migration configuration, some are addressing it sooner than others, docs were down for me when i was doing this work. Consider this resolved then.

**Review by richp405 — APPROVED** [review]

**Review by ketankelkar — APPROVED** [review]
lgtm

**Pull request merged**